test/json_texture_test.js

```diff
@@ -0,0 +1,28 @@
+var assert = require('assert');
+var configUtils = require('./utils/config');
+var testUtils = require('./utils/test');
+var fs = require('fs');
+var pkg = require('../package.json');
+
+describe('A hash map of texture info, similar to TexturePacker', function () {
+  testUtils.setInfo(configUtils.multipleSprites);
+
+  function assertValidJson() {
+    it('is valid JSON', function () {
+      var result = this.result;
+      assert.doesNotThrow(function () {
+        JSON.parse(result);
+      });
+    });
+  }
```

Comment:
don't really understand what exactly you want me to do. I'll check it in the way it is, if there are things you want to move around maybe you can do that after the merge?